include.js

```diff
@@ -6275,6 +6275,13 @@ let cards = [
     author: 'Katana',
     githubLink: 'https://github.com/katanaji'
   }
```

Comment:
There is a syntax error here. You need to separate each JavaScript object (the stuff in the curly braces) with commas.
```js
},
{
```

Reply:
Sure! it's my fault.
Thank you for your good effort ♥